chart/k8gb/values.yaml

```diff
@@ -42,6 +42,7 @@ coredns:
   servers:
   - zones:
     - zone: .
+      scheme: dns://
```

Comment:
What is that setting?

Reply:
this instructs core dns chart to listen on UDP only